### ISSUE TITLE
duplicate content

### ISSUE THREAD

**Issue by Blackbaud-LoriWeiss** [dotnet/docs#8606]
This page seems to have its content duplicated.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b0433e26-ec42-bff0-a21a-0dab9dc0d924
* Version Independent ID: b508295b-5c77-4194-3a98-b724f9fb8e2e
* Content: [FileSystem.OpenTextFileWriter Method (Microsoft.VisualBasic.FileIO)](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualbasic.fileio.filesystem.opentextfilewriter?f1url=https%3A%2F%2Fmsdn.microsoft.com%2Fquery%2Fdev15.query%3FappId%3DDev15IDEF1%26l%3DEN-US%26k%3Dk(Microsoft.VisualBasic.FileIO.FileSystem.OpenTextFileWriter);k(TargetFrameworkMoniker-.NETFramework,Version%3Dv4.6.1);k(DevLang-csharp)%26rd%3Dtrue&view=netframework-4.7.2)
* Content Source: [xml/Microsoft.VisualBasic.FileIO/FileSystem.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/Microsoft.VisualBasic.FileIO/FileSystem.xml)
* Product: **dotnet-api**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by mikkelbu** [comment]
@Blackbaud-LoriWeiss There is a

**Comment by rpetrusha** [comment]
I agree as well, @Blackbaud-LoriWeiss and @mikkelbu. It's not only that we've presented the same example for both overloads, but that we've failed to differentiate the overloads. (For example, the overload definition is OK, the definition for the first overload could either by identical or mention the append/overwrite option, and the second overload definition should definitely mention the encoding option. The goal here is that, in looking at the Overloads section, you should be able to disambiguate one overload from another; that isn't the case here. In addition, there's exception information is incomplete (at a bare minimum, both overloads throw a NullReferenceException, and possibly other exceptions).
So this documentation could use some work. I've added it to the backlog and to our community contributors' project. If either of you are interested in improving this documentation, we'd appreciate it.
